stardog/values.yaml

```diff
@@ -63,6 +63,7 @@ persistence:
 
 metrics:
   enabled: true
+  prometheusOperator: true
```

Comment:
Wouldn't enable it by default. The chart should be working with the default values and this won't be the case if there is an external dependency to the `prometheus-operator`.